manifests/syslinux.pp

```diff
@@ -1,11 +1,8 @@
 # This class will install the syslinux images into the tftp root directory.
 
-class pxe::syslinux {
-
-  include pxe::params
+class pxe::syslinux inherits pxe {
```

Comment:
I'm not keen on this style, and I think the missing local variable is causing the tests to fail.